helix-term/src/ui/popup.rs

```diff
@@ -296,32 +318,30 @@ impl<T: Component> Component for Popup<T> {
         cx.scroll = Some(scroll);
 
         // clear area
-        let background = cx.editor.theme.get("ui.popup");
-        surface.clear_with(area, background);
-
-        let render_borders = cx.editor.popup_border();
-
-        let inner = if self
-            .contents
-            .type_name()
-            .starts_with("helix_term::ui::menu::Menu")
-        {
-            area
+        let background = if is_menu {
+            // TODO: consistently style menu
+            cx.editor
+                .theme
+                .try_get("ui.menu")
+                .unwrap_or_else(|| cx.editor.theme.get("ui.text"))
         } else {
-            area.inner(&self.margin)
+            cx.editor.theme.get("ui.popup")
         };
+        surface.clear_with(area, background);
 
-        let border = usize::from(render_borders);
+        let mut inner = area;
         if render_borders {
+            inner = area.inner(&Margin::all(1));
```

Comment:
Yeah same here. I k kw the Paddington isn't ideal with borders but for docs and similar stuff I actually think the Paddington is less bad considering that they have a bunch of Paddington already.

I just don't want to add menu based special cases. In my followup PR I will remove the extra üadding from all the popups but I want to keep it simple here

Reply:
The problem is not just visual (having extra padding), but also that we cant scroll to the end of a popup if we do this for every popup

Reply:
But if its not a big issue then i dont mind having it fixed in later PR, i will run helix with my changes until then

Reply:
the scrolling was actually an unrelated bug where `area` instead of `inner` was used for computing scroll. I pushed a fix for that plus some fixes to the scrollbar rendering. For anything but menu the scrollbar now looks the way you want. The non-menu popup have a bit of extra padding with border but I think that's hardly noticeable. The main regression is the visual one affecting menu. Like I said that will be fixed in a larger followup pr